src/components/Grid/Grid.jsx

```diff
@@ -284,18 +334,19 @@ const Grid = () => {
             'YYYY-MM-DD'
           )}&endDate=${dateRanges[1].format('YYYY-MM-DD')}`
         : ''
-
-    axiosWithAuth([token])
-      .get(
-        `http://localhost:8888/sauti/client/export/?currency=${currency ||
-          'USD'}${countryQuery || ''}${marketQuery || ''}${pCatQuery ||
-          ''}${pAggQuery || ''}${productQuery || ''}${dateRangeQuery}`
-      )
+    setErr(false)
+    const query = `http://localhost:8888/sauti/client/export/?currency=${currency ||
```

Comment:
should we change this to the master/staging url?

Reply:
good catch and I already pulled it staging, so make commit to staging instead